### ISSUE TITLE
[Urban Dictionary Word of the Day] ...

### ISSUE THREAD

**Issue by sming** [raycast/extensions#12254]
### Extension

https://www.raycast.com/r0hin/raycast-urbandictionary-word-of-the-day

### Description

TL;DR the “word of the day” did not show … the word of the day, but random word worked.

### Steps To Reproduce

Hi, I installed it and immediately tried the `word of the day` command and it just “refreshed” the word, without displaying its meaning etc. I then immediately next tried “random word” and it worked - it showed all the details etc.

I can provide more testing if you like, I’m an unemployed developer! Lol

Cheers,
Pete 

Config:
[Raycast 2024-05-09 01.54.21.rayconfig.zip](https://github.com/raycast/extensions/files/15258116/Raycast.2024-05-09.01.54.21.rayconfig.zip)

Shot of it not working:
<img width="746" alt="Screenshot 2024-05-09 at 1 53 00 AM" src="https://github.com/raycast/extensions/assets/1403866/15454af5-49bd-493d-8da4-2eb0dc70407a">

**Comment by r0hin** [comment]
This is intended behavior @pernielsentikaer. You can use command Define Word of the Day to get the definition.

The inline word of the day command is intended to be pinned to favorites or something like that for quick viewing.

Hope this helps
